lib/cassia/requests/connect_device.rb

```diff
@@ -0,0 +1,39 @@
+module Cassia
+  module Requests
+    class ConnectDevice
+
+      def initialize(access_token: nil, aps: '*', mac: nil)
```

Comment:
can we rename `mac` to `device` or `device_mac` so it's more specific it's the device?